eland/operations.py

```diff
@@ -1250,6 +1251,47 @@ def to_csv(  # type: ignore
         if path_or_buf is None:
             return "".join(result)
 
+    def to_json(  # type: ignore
+        self,
+        query_compiler: "QueryCompiler",
+        path_or_buf=None,
+        orient=None,
+        lines=False,
+        **kwargs,
+    ):
+        if orient == "records" and lines is True:
+            result = []
+            processed = 0
```

Comment:
lint fails because this isn't used

Reply:
```suggestion
```

Thanks! I decided last-minute to leave out the `show_progress` parameter in this method, but that of course makes this integer no longer required.